### DISCUSSION TITLE
PimcoreAssetMetadataClassDefinitionsBundle is not installed

### DISCUSSION THREAD

**Discussion by nicolaipischel** [pimcore/pimcore#8542] [Q&A / Support]
Hey,

I'm currently facing issues with the installation of the PimcoreAssetMetadataClassDefinitionsBundle.

I've followed the instructions here: https://pimcore.com/docs/asset-metadata/current/Installation.html and the installer runs successfully but after the :install step when I execute `php ./bin/console pimcore:bundle:list` it shows this...

![image](https://user-images.githubusercontent.com/15820286/112110173-d5a91680-8bb2-11eb-89f2-ffd3bbfebf3a.png)

Unfortunately I was not able to find any valuable info on google so I'm asking here.

Can anybody assist?

**Comment by sgruber-pimcore** [comment]
Hello Nicolai!

Thank you for your question!
Could you please contact me directly per Mail?
stefan.gruber[ät]pimcore.com

Kind regards,
Stefan
